pypesto/optimize/util.py

```diff
@@ -0,0 +1,41 @@
+from ..engine import Engine, SingleCoreEngine
+from ..objective import HistoryOptions
+from ..store.save_to_hdf5 import get_or_create_group
+
+import os
+import h5py
+
+
+def check_hdf5_mp(history_options: HistoryOptions,
+                  engine: Engine):
+    """
+    This function checks whether we use another engine than
+    `SingleCoreEngine` and whether we use a `Hdf5History`. If
+    that is the case, i rephrases the filename to create a hdf5 file
```

Comment:
so delete the whole sentence and only keep 
```
This function checks whether we use another engine than
    `SingleCoreEngine` and whether we use a `Hdf5History`.  Returns filename for later usage in filling hdf5 file
    otherwise return None.
```

Reply:
Also, I would not use first person for docs. Rest of the docs doesn't do that either and better stay consistent here...

Reply:
at least in this case it was fixed 👍